jerry-core/jcontext/jcontext.h

```diff
@@ -143,17 +143,29 @@ typedef struct
 } jerry_hash_table_t;
 #endif /* !CONFIG_ECMA_LCACHE_DISABLE */
 
+/**
+ * IF EXTERNAL_CONTEXT is not enable then INIT_CURRENT_INSTANCE is a empty macro to avoid warnings
+ */
+#ifndef JERRY_ENABLE_EXTERNAL_CONTEXT
+
+#define JERRY_INIT_CURRENT_INSTANCE()
+
+#endif /* !JERRY_ENABLE_EXTERNAL_CONTEXT */
+
 #ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
 
 #ifndef JERRY_GET_CURRENT_INSTANCE
 
 /**
  * Default function if JERRY_GET_CURRENT_INSTANCE is not defined.
  */
-#define JERRY_GET_CURRENT_INSTANCE() (jerry_port_get_current_instance ())
+#define JERRY_GET_CURRENT_INSTANCE() (jerry_port_get_current_instance())
```

Comment:
The space is unnecessary before the closing bracket, but needed before the opening bracket in case of function calls. Do not modify this line.